op-challenger/fault/loader.go

```diff
@@ -19,22 +19,24 @@ type ClaimFetcher interface {
 		Clock       *big.Int
 	}, error)
 	ClaimDataLen(opts *bind.CallOpts) (*big.Int, error)
+	ABSOLUTEPRESTATE(opts *bind.CallOpts) ([32]byte, error)
 }
 
 // Loader is a minimal interface for loading onchain [Claim] data.
 type Loader interface {
 	FetchClaims(ctx context.Context) ([]types.Claim, error)
+	FetchAbsolutePrestate(ctx context.Context) ([]byte, error)
 }
 
 // loader pulls in fault dispute game claim data periodically and over subscriptions.
 type loader struct {
-	claimFetcher ClaimFetcher
+	MinimalFaultDisputeGameCaller
```

Comment:
I don't think it makes sense that the loader _is_ the caller, it should have a reference to it and not directly expose all of its methods.  So I think we need to go back to assigning this field a name.